### ISSUE TITLE
Algorithm

### ISSUE THREAD

**Issue by Jianghaoalano** [pkzhou/LGM-SLAM#1]
Hello, Is this code input a segmented image?

**Comment by pkzhou** [comment]
Yes. the segmented images are saved in seg_left folders. The segmentation of the images is generated by a roadsegmentation algorithm [Kittiseg].

**Comment by chengwei0427** [comment]
Hi, @pkzhou 
Can you upload the the relate paper? 
Thank you!

**Comment by xiaomofang** [comment]
@Jianghaoalano hello，have you been able to reproduce successfully?

**Comment by pkzhou** [comment]
> Hi, @pkzhou Can you upload the the relate paper? Thank you!

The paper link can be found in Readme.

**Comment by chengwei0427** [comment]
> > Hi, @pkzhou Can you upload the the relate paper? Thank you!
> 
> The paper link can be found in Readme.

get, thanks.
